scripts/vmbuild.rb

```diff
@@ -84,16 +82,25 @@ def verify_run(output)
 directory_name    = "#{year_month_day}_#{hour_minute}"
 timestamp         = "#{year_month_day}#{hour_minute}"
 
-targets_config    = YAML.load_file(targets_file)
-name, directory, targets = targets_config.values_at("name", "directory", "targets")
+
+directory       = "upstream"
+name            = "manageiq"
```

Comment:
Thanks for commenting on that.  I now noticed the naming is not so good since it was moved from the targets.yml to here... directory and name are pretty ambiguous in this context.